modules/crr/pages/set-up-crr.adoc

```diff
@@ -32,17 +32,65 @@ Make sure TigerGraph is *not* installed with a local loopback IP such as 127.0.0
 The following setup is needed in order to enable Cross Region Replication.
 
 
+=== Back up primary data
 
-=== Backup primary data
-
-Use GBAR to xref:backup-and-restore:index.adoc[create a backup] of the primary cluster. See xref:backup-and-restore:index.adoc[Backup and Restore] on how to create a backup.
+Use GBAR to xref:backup-and-restore:index.adoc[create a backup] of the primary cluster.
+See xref:backup-and-restore:index.adoc[Backup and Restore] for how to create a backup.
 
 If you are setting up both the primary cluster and the DR cluster from scratch, you can skip Steps 1, 2, and 4 and only perform Step 3.
 
 === Restore on the DR cluster
 
 Copy the backup files from every node to every node on the new cluster.  xref:backup-and-restore:index.adoc#_restore_from_a_backup_archive[Restore the backup] of the primary cluster on the DR cluster. See xref:backup-and-restore:index.adoc[Backup and Restore] on how to restore a backup.
 
+=== Delete metadata records
+
+After the restoration, but before enabling CRR on the DR cluster, delete the oudated Kafka metadata records.
+
+Kafka provides a script called `kafka-delete-records.sh`.
+It takes a JSON configuration file to specify which records/messages on which partition, and until which offset, you want to delete.
```

Comment:
Also at here, mention that "If offset is set to `-1`, the script will delete all records in the given partition."